### PR TITLE
CODE RUB: Removed DateTime Broker from Group Post

### PR DESCRIPTION
**Pull request by GulchexrasGithub**

### CODE REVIEW

**Comment by GulchexrasGithub** [comment]
@cjdutoit Could you have a look, it works on my machine

File: Taarafo.Core.Tests.Unit/Services/Foundations/GroupPosts/GroupPostServiceTests.Validations.Modify.cs [hunk 1]
```diff
@@ -72,10 +71,6 @@ public async Task ShouldThrowValidationExceptionOnModifyIfGroupPostIsInvalidAndL
             var expectedGroupPostValidationException =
                 new GroupPostValidationException(invalidGroupPostException);
 
-            this.dateTimeBrokerMock.Setup(broker =>
```
Comment (cjdutoit) [inline Taarafo.Core.Tests.Unit/Services/Foundations/GroupPosts/GroupPostServiceTests.Validations.Modify.cs:75]: Test also needs to validate for Guid.Empty

File: Taarafo.Core.Tests.Unit/Services/Foundations/GroupPosts/GroupPostServiceTests.cs [hunk 2]
```diff
@@ -75,7 +71,7 @@ public static TheoryData MinutesBeforeOrAfter()
         private static GroupPost CreateRandomModifyGroupPost(DateTimeOffset dates)
```
Comment (cjdutoit) [inline Taarafo.Core.Tests.Unit/Services/Foundations/GroupPosts/GroupPostServiceTests.cs:71]: Change this to `private static GroupPost CreateRandomModifyGroupPost()`

Comment (cjdutoit) [inline Taarafo.Core.Tests.Unit/Services/Foundations/GroupPosts/GroupPostServiceTests.cs:71]: Change this to `private static GroupPost CreateRandomModifyGroupPost()`

**Review by cjdutoit — CHANGES_REQUESTED** [review]
Some more changes required to remove the dates

**Comment by GulchexrasGithub** [comment]
> In `Taarafo.Core.Tests.Unit/Services/Foundations/GroupPosts/GroupPostServiceTests.Validations.Modify.cs` Since this is only one invalid value, it is better to make this test `[Fact]` and assign invalid value inside the test.

image
but, have 2 invalid values not one.

**Pull request closed without merging**